src/video/scalebit.c

```diff
@@ -50,7 +50,7 @@
 static inline void stage_scale2x(void* dst0, void* dst1, const void* src0, const void* src1, const void* src2, unsigned pixel, unsigned pixel_per_row)
 {
 	switch (pixel) {
-#if defined(__x86_64__) || defined(_M_X64) || defined(__SSE2__) || (defined(_M_IX86_FP) && (_M_IX86_FP == 2))
+#if defined(__x86_64__) || defined(_M_X64) || defined(__SSE2__) || (defined(_M_IX86_FP) && (_M_IX86_FP == 2) && defined(WITH_SSE2))
```

Comment:
Pentium 2020M does of course support SSE2. 
http://ark.intel.com/products/71142
You may have issues with your compiler.
_M_IX86_FP == 2 means that MSVC options are set to compile with SSE2.
Just set properly MSVC options to compile without SSE2 it you want to